src/ashell/shell-state.c

```diff
@@ -578,17 +582,42 @@ int32_t ashell_check_control(const char *buf, uint32_t len)
     return RET_OK;
 }
 
+int32_t ashell_set_bootcfg(char *buf)
+{
+    if (!fs_exist(buf)) {
+        comms_println("File passed to cfg doesn't exist\n");
```

Comment:
This doesn't have to be a concern right now but we need to be careful about how many strings we're adding; these are all taking away ROM space. In this function about a 100 bytes from strings alone. Instead, all "file not found" errors like this across ashell should use the same string; they don't need to mention the command that called it - the user already knows that.

The two rare errors below could be error numbers instead of big strings, eventually.